Fix ``HelpFormatter`` error with Python 3.14.0b1.
